lib/client/api.go

```diff
@@ -1901,7 +1901,7 @@ func (tc *TeleportClient) applyProxySettings(proxySettings ProxySettings) error
 		// interface hostname with default k8s port as a guess.
 		default:
 			webProxyHost, _ := tc.WebProxyHostPort()
-			tc.KubeProxyAddr = fmt.Sprintf("%s:%d", webProxyHost, defaults.KubeProxyListenPort)
+			tc.KubeProxyAddr = fmt.Sprintf("%s:%d", webProxyHost, defaults.KubeListenPort)
```

Comment:
Use https://golang.org/pkg/net/#JoinHostPort so we can support IPv6. We might have something in `utils` that does this as well.

Reply:
Nice catch, done